Exiled.Events/Commands/Config/EConfig.cs

```diff
@@ -0,0 +1,49 @@
+// -----------------------------------------------------------------------
+// <copyright file="EConfig.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.Commands.Config
+{
+    using System;
+
+    using CommandSystem;
+
+    /// <summary>
+    /// The config command.
+    /// </summary>
+    [CommandHandler(typeof(RemoteAdminCommandHandler))]
```

Comment:
There's no reason for this to be done in game. Please remove it from the ra list and add a check to ensure that only the server console is using it.

Reply:
Ok, that should resolve it